plugin/completion.py

```diff
@@ -41,81 +41,68 @@
 }
 
 
-_changes_since_sent = []  # type: List[sublime.TextChange]
-_change_id = None  # type: Any
+class LspResolveDocsCommand(sublime_plugin.TextCommand):
 
+    def run(self, edit: sublime.Edit) -> None:
+        self.view.show_popup('coming soon (╯°□°)╯︵ ┻━┻', sublime.COOPERATE_WITH_AUTO_COMPLETE)
```

Comment:
@rwols, you cannot just use text like that, you have to style it `'<div style="padding: 10px;">coming soon (╯°□°)╯︵ ┻━┻</div>'` :)